masschallenge/django-accelerator
# [AC-7000] Change language around Personal Communication section in Entrepreneur Profile

### Changes introduced in [AC-7000](https://masschallenge.atlassian.net/browse/AC-7000):
- Add a migration that updates the verbose name attribute for the following fields: 
- facebook_url
- website_url/personal_website_url
- linked_in_url
- twitter_handle

### How to test
- A sibling PR is on [accelerate](https://github.com/masschallenge/accelerate/pull/2317)
- Run `make migrate` to apply migrations
- The new names should be displayed on the respective form fields on these pages
- http://localhost:8181/startup/add/
- http://localhost:8181/accounts/register/entrepreneur/
- http://localhost:8181/accounts/register/expert/
